JShellWrapper/src/test/java/JShellWrapperTest.java

```diff
@@ -49,12 +49,10 @@ void testHelloWorld() {
 
     @Test
     void testExpressionResult() {
-        evalTest(
-                """
-            eval
-            1
-            "Hello world!\"""",
-                """
+        evalTest("""
+                eval
+                1
+                "Hello world!\"""", """
```

Comment:
Did you check that the tests still work ?

Reply:
i didn't touch this file, it was reformatted after i tried the spotless:apply command

Reply:
Still try the tests please

Reply:
no issues in wrapper tests ✅